src/lib/db.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
```

Comment:
I think this file is identical to the one in the ledger. We may want to consolidate in `five-bells-shared` at some point.